CMakeLists.txt

```diff
@@ -1,50 +1,33 @@
-
-# Copyright (c) 2013-2014, EPFL/Blue Brain Project
+# Copyright (c) 2013-2015, EPFL/Blue Brain Project
 #                     Raphael Dumusc <raphael.dumusc@epfl.ch>
 
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 project(DisplayCluster)
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMake
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
                               ${CMAKE_SOURCE_DIR}/CMake/common)
 include(GitExternal)
 include(SubProject)
 
-# Versioning
 set(VERSION_MAJOR 0)
 set(VERSION_MINOR 5)
 set(VERSION_PATCH 0)
 set(VERSION_ABI 0)
 
-# Scripts
 set(PROJECT_INCLUDE_NAME dc)
-set(DISPLAYCLUSTER_DESCRIPTION "The DisplayCluster project")
-set(DISPLAYCLUSTER_ISSUES_URL "https://bbpteam.epfl.ch/project/issues/browse/DISCL")
```

Comment:
Not anymore. The default for the Jekyll-generated index is to point to the github issues page, which also makes it available for external users/contributors.